cis194/week6/daniel-deng/src/Exercise06.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# OPTIONS_GHC -fno-warn-missing-methods #-}
+
+module CIS194.Homework06.Exercise06 (fibs3) where
+
+import CIS194.Homework06.Exercise03 (Stream(..), streamToList)
+import CIS194.Homework06.Exercise04 (streamRepeat, streamMap)
+
+import Prelude as P
+
+x :: Stream Integer
```

Comment:
@shineli1984 is it a standard practice / trick to write a Haskell based DSL in this manner? Or am I thinking too much? 😂

Reply:
I'm not which manner you referring to. Can you give me more details here?